release/release_tests.yaml

```diff
@@ -2247,6 +2247,30 @@
 
   alert: default
 
+########################
+# Alpa tests
+########################
+
+- name: alpa_opt_2_7b_sanity_check
+  group: Alpa tests
+  working_dir: alpa_tests
+
+  frequency: nightly
```

Comment:
```suggestion
  working_dir: alpa_tests
  
  python: "3.9"

  frequency: nightly
```

Reply:
good to know. keep it in my sleeve for now :)